INFO 2023-11-11 00:53:22,281 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 00:53:22,281 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 00:53:28,700 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 00:53:28,700 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 45 Error when trying to create the playlist
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 45 Error when trying to create the playlist
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,716 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,716 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,716 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,716 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,717 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,717 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,717 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,717 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,731 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,731 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,731 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,731 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
ERROR 2023-11-11 00:53:29,428 log log.py 241 Internal Server Error: /spotify/users/franserr/playlists
Traceback (most recent call last):
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django_injector/apps.py", line 169, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py", line 34, in post
    persist_retrived_data(services, parsedInfo)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py", line 47, in persist_retrived_data
    persistence.image.link_images_to_artists(artists)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 99, in link_images_to_artists
    self.link_image_to_artist(image, artist_uri)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 85, in link_image_to_artist
    self.image_serivce.link_image_to_artist(artist_uri, image_url)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/image_service.py", line 80, in link_image_to_artist
    image.save()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 105, in save
    self.clean()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 96, in clean
    raise ValidationError(
django.core.exceptions.ValidationError: ['Image cannot be linked to an artist && album.']
ERROR 2023-11-11 00:53:29,428 log log.py 241 Internal Server Error: /spotify/users/franserr/playlists
Traceback (most recent call last):
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django_injector/apps.py", line 169, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py", line 34, in post
    persist_retrived_data(services, parsedInfo)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py", line 47, in persist_retrived_data
    persistence.image.link_images_to_artists(artists)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 99, in link_images_to_artists
    self.link_image_to_artist(image, artist_uri)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 85, in link_image_to_artist
    self.image_serivce.link_image_to_artist(artist_uri, image_url)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/image_service.py", line 80, in link_image_to_artist
    image.save()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 105, in save
    self.clean()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 96, in clean
    raise ValidationError(
django.core.exceptions.ValidationError: ['Image cannot be linked to an artist && album.']
INFO 2023-11-11 01:00:00,969 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/dtos/retrieval_dtos.py changed, reloading.
INFO 2023-11-11 01:00:00,969 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/dtos/retrieval_dtos.py changed, reloading.
INFO 2023-11-11 01:00:01,485 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:00:01,485 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:01:06,148 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:01:06,148 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:01:06,663 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:01:06,663 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:02:46,184 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:02:46,184 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:02:46,774 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:02:46,774 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:03:35,036 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:03:35,036 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:03:35,672 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:03:35,672 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:03:43,050 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 01:03:43,050 album_service album_service.py 42 An error occurred while creating an album:
ERROR 2023-11-11 01:03:43,925 log log.py 241 Internal Server Error: /spotify/users/franserr/playlists
Traceback (most recent call last):
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django_injector/apps.py", line 169, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py", line 34, in post
    persist_retrived_data(services, parsedInfo)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py", line 47, in persist_retrived_data
    persistence.image.link_images_to_artists(artists)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 99, in link_images_to_artists
    self.link_image_to_artist(image, artist_uri)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 85, in link_image_to_artist
    self.image_serivce.link_image_to_artist(artist_uri, image_url)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/image_service.py", line 80, in link_image_to_artist
    image.save()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 105, in save
    self.clean()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 96, in clean
    raise ValidationError(
django.core.exceptions.ValidationError: ['Image cannot be linked to an artist && album.']
ERROR 2023-11-11 01:03:43,925 log log.py 241 Internal Server Error: /spotify/users/franserr/playlists
Traceback (most recent call last):
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django_injector/apps.py", line 169, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py", line 34, in post
    persist_retrived_data(services, parsedInfo)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py", line 47, in persist_retrived_data
    persistence.image.link_images_to_artists(artists)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 99, in link_images_to_artists
    self.link_image_to_artist(image, artist_uri)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 85, in link_image_to_artist
    self.image_serivce.link_image_to_artist(artist_uri, image_url)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/image_service.py", line 80, in link_image_to_artist
    image.save()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 105, in save
    self.clean()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 96, in clean
    raise ValidationError(
django.core.exceptions.ValidationError: ['Image cannot be linked to an artist && album.']
INFO 2023-11-11 01:04:41,639 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py changed, reloading.
INFO 2023-11-11 01:04:41,639 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py changed, reloading.
INFO 2023-11-11 01:04:42,810 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:04:42,810 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:05:00,332 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:05:00,332 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:05:06,459 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 01:05:06,459 album_service album_service.py 42 An error occurred while creating an album:
WARNING 2023-11-11 01:06:28,113 log log.py 241 Not Found: /img/icon-changelink.svg
WARNING 2023-11-11 01:06:28,113 log log.py 241 Not Found: /img/icon-changelink.svg
WARNING 2023-11-11 01:06:28,115 log log.py 241 Not Found: /img/icon-addlink.svg
WARNING 2023-11-11 01:06:28,115 log log.py 241 Not Found: /img/icon-addlink.svg
WARNING 2023-11-11 01:06:30,564 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:06:30,564 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:06:30,566 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:06:30,566 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:29:36,524 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:29:36,524 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:29:36,525 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:29:36,525 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:30:12,575 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:30:12,575 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:30:12,576 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:30:12,576 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:30:17,569 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:30:17,569 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:30:17,571 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:30:17,571 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
INFO 2023-11-11 04:06:32,711 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:06:32,711 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:06:53,432 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:06:53,432 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:06:53,958 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:06:53,958 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:07:17,926 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,926 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,928 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,928 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,930 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,930 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,931 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,931 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,932 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,932 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,934 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,934 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,935 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,935 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,937 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,937 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,938 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,938 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,939 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,939 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,941 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,941 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,942 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,942 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,944 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,944 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,945 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,945 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,947 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,947 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,948 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,948 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,950 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,950 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,952 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,952 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,953 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,953 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,955 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,955 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,956 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,956 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,958 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,958 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,959 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,959 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,961 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,961 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,963 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,963 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,964 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,964 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,966 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,966 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,967 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,967 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,968 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,968 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,969 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,969 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,971 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,971 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,972 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,972 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,974 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,974 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,975 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,975 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,977 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,977 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,978 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,978 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,979 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,979 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,981 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,981 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,982 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,982 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,983 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,983 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,985 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,985 user_service user_service.py 65 User does not exist
WARNING 2023-11-11 04:09:41,768 log log.py 241 Not Found: /admin/img/icon-changelink.svg
WARNING 2023-11-11 04:09:41,768 log log.py 241 Not Found: /admin/img/icon-changelink.svg
WARNING 2023-11-11 04:09:41,769 log log.py 241 Not Found: /admin/img/icon-addlink.svg
WARNING 2023-11-11 04:09:41,769 log log.py 241 Not Found: /admin/img/icon-addlink.svg
WARNING 2023-11-11 04:09:45,168 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:09:45,168 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:09:45,170 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:09:45,170 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:09:45,187 log log.py 241 Not Found: /favicon.ico
WARNING 2023-11-11 04:09:45,187 log log.py 241 Not Found: /favicon.ico
WARNING 2023-11-11 04:10:03,968 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:10:03,968 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:10:03,969 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:10:03,969 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
